.github/workflows/cd.yaml

```
@@ -0,0 +1,72 @@
+name: Deploy to Amazon ECS
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)